SourceX/miniwin/misc_msg.cpp

```diff
@@ -583,7 +583,7 @@ bool TranslateMessage(const MSG *lpMsg)
 {
 	if (lpMsg->message == DVL_WM_KEYDOWN) {
 		int key = lpMsg->wParam;
-		unsigned mod = (DWORD)lpMsg->lParam >> 16;
+		unsigned mod = (DWORD)lpMsg->lParam / 65536;

```

Comment:
probably an actual bitshift?

Reply:
yes, mouse params store high and low words as two 16-bit integers.